webpack.config.rollup.js

```diff
@@ -0,0 +1,26 @@
+const CompressionPlugin = require('compression-webpack-plugin');
+const { rules, plugins } = require('webpack-atoms');
+
+module.exports = {
```

Comment:
what is this one for?

Reply:
I hope the previous comments answers this :)